### PR TITLE
feature/50222_Update_Delete_wallet

### PR DESCRIPTION
**Pull request by luongdm-3080**
## Related Tickets
- [#50222](https://edu-redmine.sun-asterisk.vn/issues/50222)

## WHAT (optional)
- Add toastr.
- Add action destroy, update
- Ajax delete, update

## HOW
- I edit js, locales.
- I edit wallets_controller.rb.

## Evidence (Screenshot or Video)
Screenshot from 2022-06-23 16-27-01
Screenshot from 2022-06-23 16-12-57

### CODE REVIEW

**Comment by luongdm-3080** [comment]
ready

File: app/controllers/wallets_controller.rb [hunk 1]
```diff
@@ -18,8 +23,31 @@ def create
...
+      respond_to do |format|
+        format.html
+        format.js
+      end
+      flash.now[:danger] = t ".edit_failure_message"
+    end
+  end
+
+  def destroy
+    @wallet.destroy
```
Comment (huongnt-2545) [inline app/controllers/wallets_controller.rb]: thêm xử lý khi destroy fail

**Review by huongnt-2545 — COMMENTED** [review]
commented

**Comment by luongdm-3080** [comment]
ready

File: app/views/shared/_modal_delete.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title"><%= header %></h4>
+      </div>
+      <div class="modal-body">
+        <p><%= message %></p>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-outline-danger" data-dismiss="modal"><%= link_to t("delete"), object, method: :delete, remote: :true %></button>
```
Comment (ngocvt-0484) [inline app/views/shared/_modal_delete.html.erb]: dài thì xuống dòng e nhé

File: app/views/wallets/destroy.js.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<% if flash[:success].present? %>
+  let delete<%= @wallet.id %> = document.querySelector("#wallet" + <%= @wallet.id %>);
```
Comment (ngocvt-0484) [inline app/views/wallets/destroy.js.erb]:
```suggestion
  $("#wallet" + <%= @wallet.id %>).remove();
```

**Comment by luongdm-3080** [comment]
ready

**Pull request merged**